__main__.py

```diff
@@ -10,13 +12,19 @@
 
 ticker = st.text_input('Please enter a stock ticker that you would like to learn more about:')
 
-st.write("""1. """, ticker)
-st.write("""2. """, """phrase2""")
-st.write("""3. """, """phrase3""")
-st.write("""4. """, )
-st.write("""5. """, )
+## Retrieve file path for text output file that contains phrases to be output
+phraends_folder = os.path.join(os.getcwd(), 'Phraends_Flask')
```

Comment:
In the future, you will need to refactor these four lines. I think they could be shorter, but for now, it's good to go!